python/hail/representation/annotations.py

```diff
@@ -70,3 +71,29 @@ def get(self, item, default=None):
         :returns: Value of item if found, or default value if not.
         """
         return self._attrs.get(item, default)
+
+
+@typecheck(struct=Struct)
+def to_dict(struct):
```

Comment:
making it an instance method would be good for some reasons and weird for others. I'm not totally sure.